users/userRouter.js

```diff
@@ -1,47 +1,106 @@
-const express = require('express');
+const express = require("express");
+const Users = require("./userDb");
+const Posts = require("../posts/postDb");
 
 const router = express.Router();
 
-router.post('/', (req, res) => {
-  // do your magic!
+router.post("/", validateUser, validatePost, (req, res, next) => {
```

Comment:
Why do we need to validate a post when adding a user?
Also, how do we validate a user that does not yet exist?